docker_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Setup yarn and build qmk_configurator inside the docker container.
+
+set -e
+#set -x
+
+# Install node
+. /root/.nvm/nvm.sh
+nvm install 11.12
```

Comment:
Can't this source `.nvmrc` ?

Reply:
I had an discussion with skully on this PR and I would hold off reviewing this PR for a little while, while he reworks stuff